lib/jsdom/living/nodes/HTMLInputElement-impl.js

```diff
@@ -167,6 +167,8 @@ class HTMLInputElementImpl extends HTMLElementImpl {
   _legacyPreActivationBehavior() {
     if (this.type === "checkbox") {
       this.checked = !this.checked;
+      this._legacyActivationBehaviorPreviousIndeterminateState = this.indeterminate;
```

Comment:
Can we have this property "declared" in the constructor for better state tracking?